app/services/users.js

```diff
@@ -6,3 +6,11 @@ exports.findUserByEmail = email => User.findOne({ where: { email } });
 
 exports.findAndCountAllUsers = ({ offset, limit }) =>
   User.findAndCountAll({ offset, limit, order: [['id', 'asc']] });
+
+exports.createAdminUser = userParams => {
+  const defaultParams = { ...userParams, role: 'admin' };
+  return User.findOrCreate({
+    where: { email: userParams.email },
+    defaults: defaultParams
+  }).then(([user, created]) => (created ? user : user.update(defaultParams)));
```

Comment:
You should catch the error and throw a dbError instead. I see the other functions in this file are not doing it either

Reply:
Also, remember it's a good practice to log at the start of the services